wcfsetup/install/files/acp/templates/packageSearchResultList.tpl

```diff
@@ -1,43 +1,53 @@
-{hascontent}
-	<table class="table">
-		<thead>
+<table class="table">
+	<thead>
+		<tr>
+			<th colspan="2" class="columnTitle">{lang}wcf.acp.package.name{/lang}</th>
+			<th class="columnText">{lang}wcf.acp.package.author{/lang}</a></th>
+			<th class="columnText">{lang}wcf.acp.package.license{/lang}</th>
+			<th class="columnDate">{lang}wcf.acp.package.packageDate{/lang}</a></th>
+			
+			{event name='headColumns'}
+		</tr>
+	</thead>
+	
+	<tbody>
+		<tr>
+			<td colspan="5"><small>{lang count=$trustedSources|count}wcf.acp.package.search.result.trusted{/lang}</small></td>
+		</tr>
+		{foreach from=$trustedSources item=$package}
+			<tr class="jsPackageRow">
+				<td class="columnIcon">
+					<a href="#" class="jsInstallPackage" data-confirm-message="{lang __encode=true}wcf.acp.package.install.confirmMessage{/lang}" data-package="{$package->package}" data-package-version="{$package->getAccessibleVersion()->packageVersion}"><span class="icon icon16 fa-plus jsTooltip" title="{lang}wcf.acp.package.button.installPackage{/lang}"></span></a>
+				</td>
+				<td class="columnTitle" title="{$package->packageDescription}">
+					<div class="packageSearchName">{$package->packageName} <span class="packageSearchVersion">{$package->getAccessibleVersion()->packageVersion}</span></div>
+					<span class="packageSearchPackage">{$package->package}</span>
+				</td>
+				<td class="columnText">{if $package->authorURL}<a href="{$package->authorURL}" class="externalURL">{$package->author}</a>{else}{$package->author}{/if}</td>
+				<td class="columnText">{if $package->getAccessibleVersion()->licenseURL}<a href="{$package->getAccessibleVersion()->licenseURL}" class="externalURL">{$package->getAccessibleVersion()->license}</a>{else}{$package->getAccessibleVersion()->license}{/if}</td>
+				<td class="columnDate">{@$package->getAccessibleVersion()->packageDate|time}</td>
+			</tr>
+		{/foreach}
+		{hascontent}
 			<tr>
-				<th colspan="2" class="columnTitle">{lang}wcf.acp.package.name{/lang}</th>
-				<th class="columnText">{lang}wcf.acp.package.author{/lang}</a></th>
-				<th class="columnText">{lang}wcf.acp.package.version{/lang}</th>
-				<th class="columnText">{lang}wcf.acp.package.license{/lang}</th>
-				<th class="columnDate">{lang}wcf.acp.package.packageDate{/lang}</a></th>
-				
-				{event name='headColumns'}
+				<td colspan="5"><small>{lang count=$thirdPartySources|count}wcf.acp.package.search.result.thirdParty{/lang}</small></td>
 			</tr>
-		</thead>
-		
-		<tbody>
 			{content}
-				{foreach from=$packageUpdates item=$package}
+				{foreach from=$thirdPartySources item=$package}
```

Comment:
When searching for a package, the following error is thrown (`PackageUpdateAction::search` does not assign the variable to the template, if no package are found):  
```
Undefined index: trustedSources


Stacktrace:

#0 /Applications/MAMP/htdocs/hurricane/acp/templates/compiled/0_wcf_1_packageSearchResultList.php(30): wcf\system\WCF::handleError(8, 'Undefined index...', '/Applications/M...', 30, Array)
#1 /Applications/MAMP/htdocs/hurricane/lib/system/template/TemplateEngine.class.php(321): include('/Applications/M...')
#2 /Applications/MAMP/htdocs/hurricane/lib/system/template/TemplateEngine.class.php(549): wcf\system\template\TemplateEngine->display('packageSearchRe...', 'wcf', false)
#3 /Applications/MAMP/htdocs/hurricane/lib/data/package/update/PackageUpdateAction.class.php(99): wcf\system\template\TemplateEngine->fetch('packageSearchRe...')
#4 [internal function]: wcf\data\package\update\PackageUpdateAction->search()
#5 /Applications/MAMP/htdocs/hurricane/lib/data/AbstractDatabaseObjectAction.class.php(204): call_user_func(Array)
#6 /Applications/MAMP/htdocs/hurricane/lib/action/AJAXProxyAction.class.php(75): wcf\data\AbstractDatabaseObjectAction->executeAction()
#7 /Applications/MAMP/htdocs/hurricane/lib/action/AJAXInvokeAction.class.php(94): wcf\action\AJAXProxyAction->invoke()
#8 /Applications/MAMP/htdocs/hurricane/lib/action/AbstractAction.class.php(47): wcf\action\AJAXInvokeAction->execute()
#9 /Applications/MAMP/htdocs/hurricane/lib/action/AJAXInvokeAction.class.php(61): wcf\action\AbstractAction->__run()
#10 /Applications/MAMP/htdocs/hurricane/lib/system/request/Request.class.php(83): wcf\action\AJAXInvokeAction->__run()
#11 /Applications/MAMP/htdocs/hurricane/lib/system/request/RequestHandler.class.php(96): wcf\system\request\Request->execute()
#12 /Applications/MAMP/htdocs/hurricane/acp/index.php(9): wcf\system\request\RequestHandler->handle('wcf', true)
#13 {main}```

Reply:
Thanks, fixed.